### PR TITLE
Fake resolutions

### PR DESCRIPTION
**Pull request by sbesson**
See https://trello.com/c/coh3IxDi/34-fake-resolution-examples-and-doc

This expands the fake images page of the technical documentation to cover the new sub-resolutions keys introduced in Bio-Formats 6

### CODE REVIEW

File: sphinx/developers/generating-test-images.rst [hunk 1]
```diff
@@ -119,6 +133,23 @@ to create where each region of interest contains a single shape of the input
...
+.. versionadded:: 6.0.0
+
+
+To generate a fake file containing sub-resolutions::
+
+    touch "pyramid1&sizeX=20000&sizeY=10000&resolutions=8.fake"
+    touch "pyramid2&sizeX=20000&sizeY=10000&resolutions=4&resolutionScale=4.fake"
+
+The ``resolutions`` and ``resolutionScale`` specify the number of
+sub-resolutions for with each plane and the downsampling factor between
```
Comment (melissalinkert) [inline sphinx/developers/generating-test-images.rst]: Remove ```with```?

File: sphinx/developers/generating-test-images.rst [hunk 2]
```diff
@@ -2,20 +2,34 @@ Generating test images
 ======================
 
 Sometimes it is nice to have a file of a specific size or pixel type for
-testing. To generate an image file (that contains a gradient image):
+testing. Bio-Formats defines an internal format used for generating test images. Test images recognized by Bio-Formats:
```
Comment (mtbc) [inline sphinx/developers/generating-test-images.rst]: maybe wrap this line

File: sphinx/developers/generating-test-images.rst [hunk 3]
```diff
@@ -2,20 +2,34 @@ Generating test images
...
+
+- must have an extension of type `.fake`
+- must encode the image metadata using key-value pairs separated by `=` either
```
Comment (joshmoore) [inline sphinx/developers/generating-test-images.rst]: I'd put this after the definition of companion file

**Comment by sbesson** [comment]
Given the feedback, thoughts about getting this in and opening a follow-up PR to wrap the line, restructure the key/value pairs table  and rework the Global Metadata/Series Metadata section?

**Comment by joshmoore** [comment]
WFM.

**Pull request merged**